docs/metadata/configure-audit-logging.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Configure Audit Logging
+id: auditLogging
+description: Prophecy Audit logs available to be exported in S3
+sidebar_position: 6
+tags:
+  - Audit Logs
+---
+
+Prophecy provides access to audit logs of activities performed by Prophecy users, allowing your enterprise to monitor detailed usage patterns.
+User can configure a S3 bucket to sync these events from Prophecy to their environment.
+
+## Configure S3 bucket for logs
+
+1. Create/reuse an AWS IAM user who requires access to Prophecy audit logs. There is no need to attach any policies to this user.
+2. Share the ARN of the AWS user with Prophecy, inorder to grant permissions to access the audit events stored in Prophecy S3 bucket. The ARN should be of the format `arn:aws:iam::{{Customer-Account-ID}:user/{Customer-User}` with the actual customer details. Eg: `arn:aws:iam::123456789012:user/bob`.
+3. Reach out to Prophecy at [contact us](mailto:success@Prophecy.io) with above details to enable this in your account.
+4. Prophecy would provide the required access and communicate back the S3 bucket URL that can be used to access the events.
+5. You may now be able to access this bucket over an application (or) AWS CLI using the above user credentials.
+6. Example of AWS CLI access is given below. Use the below command to get the latest events bucket locally. You may use similar mechanism to access it from your applications.
```

Comment:
Could @manjunathprophecy put together a summary of setting up Splunk integration on s3? Or maybe link to some tutorial. I'm sure Workday will need help.

Reply:
We have not tried Yet this Audit log by integrating into Splunk ,  We will setup one for within our env. and later we can document ?  and add it here